azure/notebooks/remote-explanation/azure-gpu-shap.ipynb

```diff
@@ -31,7 +24,7 @@
     "# apt-get install -y python3-dev && \\\n",
     "# pip install azureml-core && \\\n",
     "# pip install azureml-interpret && \\\n",
-    "# pip install interpret-community==0.18 && \\\n",
+    "# pip install -e git+https://github.com/interpretml/interpret-community.git#egg=interpret_community\\&subdirectory=python && \\\n",
```

Comment:
This should be fine for now, but let's update as soon as interpret-community is upgraded  to 0.19